lib/matplotlib/tests/test_legend.py

```diff
@@ -969,13 +969,12 @@ def test_legend_pathcollection_labelcolor_markfacecolor_cmap():
     # test the labelcolor for labelcolor='markerfacecolor' on PathCollection
     # with colormaps
     fig, ax = plt.subplots()
-    facecolors = mpl.cm.viridis(np.random.rand(10))
+    colormap = mpl.cm.viridis(np.random.rand(10))
```

Comment:
The naming is misleading. `viridis` is the colormap. You call it and get a list of colors as a result.
```suggestion
    colors = mpl.cm.viridis(np.random.rand(10))
```